test/e2e/storage/persistent_volumes-local.go

```diff
@@ -774,10 +774,16 @@ func checkPodEvents(config *localTestConfig, podName string, ep *eventPatterns)
 	}.AsSelector().String()
 	options := metav1.ListOptions{FieldSelector: selector}
 	events, err := config.client.CoreV1().Events(config.ns).List(options)
-	Expect(err).NotTo(HaveOccurred())
-	Expect(len(events.Items)).NotTo(Equal(0))
+	if err != nil {
```

Comment:
we can use this way to add error messages? 
Expect(err).NotTo(HaveOccurred(), fmt.Sprintf(...))